solr/solr-ref-guide/modules/upgrade-notes/pages/major-changes-in-solr-9.adoc

```diff
@@ -68,6 +68,18 @@ In Solr 8, it was possible to add docValues to a schema without re-indexing via
 Due to changes in Lucene 9, that isn't possible any more.
 
 == Solr 9.5
+
+=== SchemaVersion upgraded to 1.7
+Starting with Solr 9.5, the default schemaVersion is 1.7.
+With the new schema version, all fields and fieldTypes that support docValues (StrField, *PointField, BoolField, etc.)
+will have docValues enabled by default.
+
+In order to upgrade from schemaVersion 1.6 to 1.7, a re-index of all data will generally be required.
```

Comment:
Should we mention that the re-index can't be in-place (for Solr 9 indices at least)?

Reply:
Why can't the re-index be in-place? Or do you mean atomic updates?

Reply:
Lucene 9 enforces a strict schema. I believe adding docValues to an existing fields will not be allowed. See LUCENE-9334

Reply:
Good catch, we should definitely mention that.